src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LambdaHandler.java

```diff
@@ -83,7 +98,8 @@ public void checkIndentation() {
         if (parent.getType() != TokenTypes.SWITCH_RULE
                 && getLineStart(firstChild) == expandedTabsColumnNo(firstChild)) {
             final IndentLevel level = getIndent();
-            if (!level.isAcceptable(expandedTabsColumnNo(firstChild))) {
+            if (isAcceptableIndent(firstChild, level)) {
+                isLambdaCorrectlyIndented = false;
```

Comment:
Field name also doesn't make sense to me.

You set lambda to not be correctly idented when the indent is acceptable... I see other instances where it is set to false just because of the parent and nothing to do with the indentation.

Reply:
@rnveach  Yes, there is really a blunder in the naming... It is flipping actual logic...
The method name should be `ifNonAcceptableIndent`.

Reply:
changes made.